pkg/storage/s3/s3_test.go

```diff
@@ -1628,6 +1625,8 @@ func TestS3Dedupe(t *testing.T) {
 
 					time.Sleep(10 * time.Second)
 
+					taskScheduler.Shutdown()
```

Comment:
I use it before calling Stat() and GetBlobContent() because shutdown makes sure all tasks finished their work before exiting.

I can put both a defer and leave that Shutdown like it is.